python/_tskitmodule.c

```diff
@@ -8765,6 +8765,26 @@ Tree_get_children(Tree *self, PyObject *args)
     return ret;
 }
 
+static PyObject *
+Tree_depth(Tree *self, PyObject *args)
+{
+    PyObject *ret = NULL;
+    unsigned int depth;
+    int node, err;
+
+    if (Tree_get_node_argument(self, args, &node) != 0) {
+        goto out;
+    }
+    err = tsk_tree_depth(self->tree, node, &depth);
+    if (ret != 0) {
+        handle_library_error(err);
+        goto out;
+    }
+    ret = Py_BuildValue("I", depth);
```

Comment:
As we're changing depth to a (tsk_size_t), it'll be good to cast this to an unsigned int now, like
```c
ret = Py_BuildValue("I", (unsigned int) depth);
```
we play a bit loosey-goosey with the types here, as the Python C API doesn't have converter strings for specified sizes like uint32_t etc.